.pre-commit-config.yaml

```
@@ -10,7 +10,7 @@ repos:
     hooks:
       - id: roxygenize
         additional_dependencies:
-          - formatters
+          - insightsengineering/formatters
```

```suggestion
          - formatters
```
I do not think we need this right?

---

we did for precommit to work locally, I think, but we don't ultimately need that change, no